lightning/src/ln/channelmanager.rs

```diff
@@ -6393,137 +6393,155 @@ where
 		should_persist
 	}
 
-	fn process_forward_htlcs(
-		&self, short_chan_id: u64, pending_forwards: &mut Vec<HTLCForwardInfo>,
-		failed_forwards: &mut Vec<FailedHTLCForward>,
+	/// Fail the list of provided HTLC forwards because the channel they were to be forwarded over does no longer exist.
+	#[rustfmt::skip]
```

Comment:
Added skip to make it easy to verify that the code didn't change. Use `--ignore-space-change`